ts/components/session/settings/SessionSettingListItem.tsx

```diff
@@ -88,43 +88,43 @@ export class SessionSettingListItem extends React.Component<Props, State> {
             />
           )}
 
-          {type === SessionSettingType.Slider && title === 'Message TTL' ? (
+          {type === SessionSettingType.Slider &&
+          content.sliderCategory === 'messageTTLSlider' ? (
             <div className="slider-wrapper">
               <Slider
-                dots={true}
-                step={6}
-                min={12}
-                max={96}
+                dots={content.dotsEnable}
+                step={content.step}
+                min={content.min}
+                max={content.max}
                 defaultValue={currentSliderValue}
                 onAfterChange={sliderValue => {
                   this.handleSlider(sliderValue);
                 }}
               />
-              
+
               <div className="slider-info">
                 <p>{`${currentSliderValue} Hours`}</p>
               </div>
             </div>
-          ):type === SessionSettingType.Slider && title === "Zoom Factor" ? (
+          ) : type === SessionSettingType.Slider &&
+          content.sliderCategory === 'zoomFactorSlider' ? (
```

Comment:
This is unnecessary, we shouldn't be checking `sliderCategory` on any slider since it just duplicates code.

 What needs to be done instead is to create a field in `content` which is a function which returns what the slider info needs to display.

E.g
```
content: {
...,
info: (value) => `Zoom Level: ${value}%`
}
```
```

Reply:
all the suggested change has been made;
request for further review. Thanks.